compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.kt

```diff
@@ -345,6 +356,23 @@ internal class MetalRedrawer(
             }
         }
     }
+
+    private fun configureMetalHUDIfNeeded() {
+        if (available(OS.Ios to OSVersion(16))) {
+            val entry = NSBundle
+                .mainBundle
+                .objectForInfoDictionaryKey("MetalHudEnabled") as? NSNumber
+
+            if (entry?.boolValue == true) {
+                dispatch_async(dispatch_get_main_queue()) {
```

Comment:
Do we need to dispatch? The constructor should be called in the main thread

Reply:
Oof, that's a copy-paste error. Thanks. Removed.